Update Dev Kit sample testing process (#915)
OfficeDev/Office-Add-in-samples: ADD_SAMPLE_INTO_DEV_KIT.md

## Testing

1. Install the latest Office Add-ins Development Kit, make sure the version is equal or higher than 1.0.1.

> 2. Set the environment variable `OTK_SAMPLE_CONFIG_BRANCH` to `dev` or other branch with a valid [sample-config.json](./.config/sample-config.json) of this repo on your development machine. Setting this variable will force sample gallery to use the config in the dedicated branch instead of `main`.

Reviewer: ```suggestion
2. Set the environment variable `OTK_SAMPLE_CONFIG_BRANCH` to `dev` or other branch with a valid [sample-config.json](./.config/sample-config.json) of this repo on your development machine. Setting this variable will force the sample gallery to use the configuration in the dedicated branch instead of `main`.
```

> 3. Open Dev Kit. Select `View Samples` to view sample gallery and do testing. 

Reviewer: ```suggestion
3. Open the Development Kit. Select `View Samples` to view sample gallery and test. 
```
